Engine/GameObjects/Containers/Container.cs

```diff
@@ -2,21 +2,33 @@
 using System.Collections.Generic;
 using System.Text;
 using Text_Based_Adventure.Engine.GameObjects.Items;
+using Text_Based_Adventure.Engine.GameObjects.SkillChecks;
 using Text_Based_Adventure.GameObjects;
 
 namespace Text_Based_Adventure.Engine.GameObjects.Containers
 {
     public class Container : Item
     {
         public List<Item> Items;
-        public LockObject LockObject;
+        public SkillCheck Lock;
         public List<string> ItemStrings;
 
         public Container()
         {
             Items = new List<Item>() { };
         }
 
+        public List<Item> UnloadAllItems()
+        {
+            List<Item> returnedItems = new List<Item>();
+            foreach (var item in Items)
```

Comment:
You can shorten the `foreach` loop to `returnedItems = Items.ToList();` This creates a new copy of the list.